source/scpp/extra/DUtils.cpp

```diff
@@ -147,15 +147,19 @@ CPPOBJECTINST(std::set<NodeID>);
 CPPOBJECTINST(std::set<SCPBallot>);
 CPPOBJECTINST(std::set<unsigned int>);
 
-CPPOBJECTINST(std::vector<unsigned char>);
-CPPOBJECTINST(std::vector<Value>);
-CPPOBJECTINST(std::vector<unsigned long>);
-CPPOBJECTINST(std::vector<unsigned long long>);
-CPPOBJECTINST(std::vector<SCPQuorumSet>);
-CPPOBJECTINST(std::vector<SCPEnvelope>);
-CPPOBJECTINST(std::vector<Slot::HistoricalStatement>);
 CPPOBJECTINST(stellar::SCPStatement::_pledges_t);
 
+#define CPPVECINST(T) template class T; \
+                      CPPOBJECTINST(T)
```

Comment:
Why this BTW ?

Reply:
to force generation of all `std::vector` symbols, otherwise only our wrappers (`defaultCtorCPPObject` et al) get generated.